src/System.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -9,9 +9,10 @@ namespace System.Data.SqlClient.ManualTesting.Tests
     public static class LocalDBTest
     {
         private static bool IsLocalDBEnvironmentSet() => DataTestUtility.IsLocalDBInstalled();
+        private static bool AreConnectionStringsSetup() => DataTestUtility.AreConnStringsSetup();
 
         [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // No Registry support on UAP
-        [ConditionalFact(nameof(IsLocalDBEnvironmentSet))]
+        [ConditionalFact(nameof(IsLocalDBEnvironmentSet),nameof(AreConnectionStringsSetup))]
```

Comment:
You don't need to check for Conn str in  localdb test. It has its own Conn str in the test. Same for following tests

Reply:
Reverted back the changes in LocalDBTest.cs